components/brave_wallet_ui/page/wallet_page.html

```diff
@@ -3,12 +3,23 @@
   <head>
   <base href="/">
   <meta charset="utf-8">
-  <meta name="viewport" content="width=device-width">
+  <if expr="is_ios">
+    <meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=1">
+  </if>
+  <if expr="not is_ios">
+    <meta name="viewport" content="width=device-width">
+  </if>
```

Comment:
Could you please add a comment why we need `initial-scale=1, maximum-scale=1` for ios specifically